### PR TITLE
add range_theta kwarg

### PR DESCRIPTION
**Pull request by nicolaskruchten**
Closes https://github.com/plotly/plotly_express/issues/160

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
This enables this kind of thing:

image

**Comment by emmanuelle** [comment]
Thank you! Would you mind updating also `doc/python/polar-chart.md` with an example?

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,10 +57,10 @@ jobs:
       - checkout
       - run:
           name: Install tox
-          command: 'sudo pip install tox'
+          command: "sudo pip install tox"
       - run:
           name: Test with tox
-          command: 'cd packages/python/plotly; tox -e py36-core'
+          command: "cd packages/python/plotly; tox -e py36-core"
```
Comment (emmanuelle) [inline .circleci/config.yml]: just curious to know why you made these changes? In cases variables had to be interpreted within the strings? Asking only for my own enlightenment :-)

**Comment by emmanuelle** [comment]
:dancer: once the CI is green!

File: .circleci/config.yml (hunk 1 above)
Comment (nicolaskruchten) [reply .circleci/config.yml]: oh umm I didn't intentionally change anything there! I must have an autoformatter

**Pull request merged**